### PR TITLE
Remove `<br />` tags from legacy ui error messages

### PR DESCRIPTION
**Pull request by jmmacdo4**
By remove `<br />` tags from legacy ui error messages, we can turn HTML escaping on and avoid the possibility of XSS attacks.

### CODE REVIEW

**Comment by jmmacdo4** [comment]
@isears Here is the `<br />` to comma rework PR.

File: omod/src/main/java/org/openmrs/web/controller/maintenance/ImplementationIdFormController.java [hunk 1]
```diff
@@ -23,6 +23,7 @@
 import org.springframework.web.servlet.ModelAndView;
 import org.springframework.web.servlet.mvc.SimpleFormController;
 import org.springframework.web.servlet.view.RedirectView;
+import org.apache.commons.lang.StringEscapeUtils;
```
Comment (HerbertYiga) [inline omod/src/main/java/org/openmrs/web/controller/maintenance/ImplementationIdFormController.java:26]: do we need this change?

Comment (jmmacdo4) [reply omod/src/main/java/org/openmrs/web/controller/maintenance/ImplementationIdFormController.java]: No, thanks Herbert I fixed that.

**Review by HerbertYiga — APPROVED** [review]
looks good to me  cc @isears

**Review by isears — APPROVED** [review]
Ok, this looks good. Thanks @jmmacdo4 and thanks for reviewing @HerbertYiga !

Also looks like the merge conflict with #138 can be resolve automatically

**Pull request merged**